Core/include/Acts/Detector/detail/CylindricalDetectorHelper.hpp

```diff
@@ -29,6 +29,26 @@ class Portal;
 namespace detail {
 namespace CylindricalDetectorHelper {
 
+/// @brief Loweset level connection in R
+///
+/// @param keepCoverVolume the volume where the cover is kept
+/// @param wasteCoverVolume the volume where the cover is binned
+void fuseInR(DetectorVolume& keepCoverVolume, DetectorVolume& wasteCoverVolume);
+
+/// @brief Lowest level connection in Z
+///
+/// @param keepEndplateVolume the volume where the cover is kept
+/// @param wasteEndplateVolume the volume where the cover is binned
+void fuseInZ(DetectorVolume& keepEndplateVolume,
+             DetectorVolume& wasteEndplateVolume);
+
+/// @brief Lowest level connection in Phi
+///
+/// @param keepSectorVolume the volume where the sector is kept
+/// @param wasteSectorVolume the volume where the sector is binned
+void fuseInPhi(DetectorVolume& keepSectorVolume,
+               DetectorVolume& wasteSectorVolume);
+
```

Comment:
This is used now by this class and the grid builder, hence I encaspulated it into convenience functions.